apps/platform-app/src/lib/langchain.ts

```diff
@@ -0,0 +1,194 @@
+'use server'
```

Comment:
**issue:** This file should not be in `src/lib` as it's domain specific logic. Let's place it in new directory in `app/(profile)/_workflows/profile-evaluation.workflow.ts`